components/brave_rewards/resources/ui/components/pageWallet.tsx

```diff
@@ -67,6 +69,43 @@ class PageWallet extends React.Component<Props, State> {
     })
   }
 
+  onModalBackupOnCopy = (backupKey: string) => {
+    const success = clipboardCopy(backupKey)
+    // TODO(jsadler) possibly flash a message that copy was completed
+    console.log(success ? 'Copy successful' : 'Copy failed')
+  }
+
+  onModalBackupOnPrint = (backupKey: string) => {
+    let printWindow = window.open(backupKey)
+    if (printWindow) {
+      printWindow.document.open('plain/text')
+      printWindow.document.write(this.constructBackup(backupKey))
+      printWindow.document.close()
+      printWindow.focus()
+      printWindow.print()
+      printWindow.close()
+    }
+  }
+
+  constructBackup = (backupKey: string) => {
```

Comment:
[minor] is it really necessary to define this since it's just an alias for `utils.constructBackupString`?

Reply:
Removed.